src/main/java/chess/IMovable.java

```diff
@@ -2,11 +2,8 @@
 import java.util.List;
 
 interface IMovable {
-    void move(Piece target);
-
-    boolean isPosibleMove(Piece target);
-
-    List<Piece> getPosibleMoves(ChessBoard ch);
+    boolean isPossibleMove(Position target, ChessBoard chess);
+    List<Position> getPossibleMoves(ChessBoard ch);
```

Comment:
These two methods don't need to pass a ChessBoard as an argument, because of all Pieces have a reference to main ChessBoard. So, you also need to change all classes which implement this interface.

Reply:
On line 5 it is necessary because we are sending the chessboard in the Method Move from Piece.java

Reply:
You are right, Change made can you review it?